### DISCUSSION TITLE
Update to Angular 13

### DISCUSSION THREAD

**Discussion by tomii8** [DanielNetzer/ngx-diagrams#142] [General]
Hi @DanielNetzer @DanielNetzeriAm, do you maybe have in plan to update angular wrapper to Angular 13? On our project we use full Ivy compilation and compiler is complaining on @rxzu/angular library.

**Comment by DanielNetzer** [comment]
@tomii8 You can find the updated version with support for Angular 13 at https://github.com/netzerstudio/rxzu
